doc/man/scons.xml

```diff
@@ -2901,47 +2889,13 @@ env.Program(target='bar', source=env.Split('bar.c foo.c'))
 env.Program('bar', source='bar.c foo.c'.split())
 </programlisting>
 
-<para>
-Sources and targets can be specified as a scalar or as a list,
-composed of either strings or nodes (more on nodes below).
-When specifying path strings,
-&Python; follows the POSIX pathname convention:
-if a string begins with the operating system pathname separator
-(on Windows both the slash and backslash separator are accepted,
```

Comment:
Is this info included elsewhere in your modified version? (slash and backslash both accepted on windows for dirsep?)

Reply:
Been a while... I think it just felt redundant. Didn't think we needed to be defining what an absolute path is.